app/models/link.rb

```
@@ -1352,14 +1352,18 @@ def as_json_for_api(options)
 
       ppp_factors = purchasing_power_parity_enabled? ? options[:preloaded_ppp_factors] || PurchasingPowerParityService.new.get_all_countries_factors(user) : nil
 
+      preloaded_tags = tags.loaded? ? tags : tags.includes(:product_taggings)
```

RuboCop (1.75.5)

[warning] Useless assignment to variable - `preloaded_tags`. Did you mean `preloaded_prices`?

(Lint/UselessAssignment)